framework/src/modules/chain/components/storage/entities/account.js

```diff
@@ -150,6 +142,37 @@ class ChainAccount extends AccountEntity {
 		);
 	}
 
+	/**
+	 * Parse account data and parse in default values
+	 * @param {Array[Object}] | Object} data raw database account data
+	 * @return {Array[Object]} Parsed accounts
+	 */
+	static _sanitizeCreateData(data) {
+		let accounts;
+		if (Array.isArray(data)) {
+			accounts = data.map(item => ({
+				...item,
+			}));
+		} else if (typeof data === 'object') {
+			accounts = [
+				{
+					...data,
+				},
+			];
+		}
+
+		accounts = accounts.map(account => {
+			const parsedAccount = _.defaults(account, defaultCreateValues);
+			parsedAccount.asset = parsedAccount.asset
+				? JSON.stringify(parsedAccount.asset)
```

Comment:
You don't need to stringify it manually, underlaying postgresql adapter must handle it. Please test it by just passing as an object.

Reply:
Correct catch, thanks!